wcc-contentful-app/app/helpers/wcc/contentful/app/section_helper.rb

```diff
@@ -53,11 +53,12 @@ def markdown(text)
     }
 
     renderer = ::WCC::Contentful::App::CustomMarkdownRender.new(options)
-    remove_markdown_href_class_syntax(raw_classes, text)
     markdown = ::Redcarpet::Markdown.new(renderer, extensions)
 
     content_tag(:div,
-      markdown.render(text).html_safe,
+      CGI.unescapeHTML(markdown.render(
```

Comment:
Do you need to unescape? `html_safe` could be enough?

Reply:
Yeah, the html_safe had been there the entire time, but the unescape was the trick. Check it out:
```
text
#=> [Children's](https://watermark.formstack.com/forms/childrensvolunteer)
remove_markdown_href_class_syntax(raw_classes, text)
#=> [Children's](https://watermark.formstack.com/forms/childrensvolunteer)
markdown.render(remove_markdown_href_class_syntax(raw_classes, text))
#=> <p><a target="_blank" href="https://watermark.formstack.com/forms/childrensvolunteer">Children&amp;#39;s</a></p>
markdown.render(remove_markdown_href_class_syntax(raw_classes, text)).html_safe
#=> <p><a target="_blank" href="https://watermark.formstack.com/forms/childrensvolunteer">Children&amp;#39;s</a></p>
CGI.unescapeHTML(markdown.render(remove_markdown_href_class_syntax(raw_classes, text))).html_safe
#=> <p><a target="_blank" href="https://watermark.formstack.com/forms/childrensvolunteer">Children&#39;s</a></p>
```